packages/astro/src/@types/astro.ts

```diff
@@ -444,15 +444,22 @@ export interface AstroUserConfig {
 	 * @name base
 	 * @type {string}
 	 * @description
-	 * The base path to deploy to. Astro will build your pages and assets using this path as the root. Currently, this has no effect during development.
+	 * The base path you're deploying to. Astro will match this pathname during development so that your development experience matches your build environment as closely as possible. You can access this value in your app via `import.meta.env.BASE_URL`.
```

Comment:
```suggestion
	 * The base path you're deploying to. Astro will match this pathname during development so that your development experience matches your build environment. 
```